code/src/main/java/com/google/sps/data/Location.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.data;
+
+/* Store a location based on longitude/latitude coordinates and
+ * the time interval when the user was present
+ */
+public class Location {
+
+  long longitudeE7;
```

Comment:
Getters/Setters? Or public fields?

Reply:
I was thinking public fields, however this discussion is covered by the first comment